tutorials/express/express-json.md

```diff
@@ -0,0 +1,58 @@
+The [`app.use()`](/tutorials/express/app-use) function adds a new middleware to the app.
+Essentially, whenever a request hits your backend, the `app.use()` functions will trigger.
```

Comment:
"the `app.use()` functions will trigger" -> "Express will execute the functions you passed to `app.use()` in order"